plejd/PlejdApi.js

```diff
@@ -369,30 +370,46 @@ class PlejdApi {
 
           this.deviceRegistry.addOutputDevice(outputDevice);
         }
-      }
-
-      // What should we do with inputs?!
-      // if (outputDevice.typeName === 'WPH-01') {
-      //   // WPH-01 is special, it has two buttons which needs to be
-      //   // registered separately.
-      //   const inputs = this.siteDetails.inputAddress[deviceId];
-      //   const first = inputs[0];
-      //   const second = inputs[1];
-
-      //   this.deviceRegistry.addPlejdDevice({
-      //     ...outputDevice,
-      //     id: first,
-      //     name: `${device.title} left`,
-      //   });
-
-      //   this.deviceRegistry.addPlejdDevice({
-      //     ...outputDevice,
-      //     id: second,
-      //     name: `${device.title} right`,
-      //   });
-      // } else {
-      //   this.deviceRegistry.addPlejdDevice(outputDevice);
-      // }
+      } else {
```

Comment:
This shouldn't be an "else" I think. Devices should be allowed to have both inputs and outputs.
However, we do know that Plejds tend to not send BLE commands for input devices WITH outputs. So... 

Should we after all use the "else" (despite it being less "generic)? Or should we do some "if (input AND [someothercondition]"? Or something else? Can you see in the API that the input will be sent over BLE?

Reply:
I was a bit puzzled over this. In the end I decided that the most generic  way to identify a "pure" input device was when it did not have an output address.
But, there can be such conditions - eg. when a device has no load connected to it. Hence, I added the following code. See your next comment.